tests/csrf_tests/tests.py

```diff
@@ -99,18 +114,18 @@ def _read_csrf_cookie(self, req, resp):
         """
         raise NotImplementedError('This method must be implemented by a subclass.')
 
-    def _get_request(self, method=None, cookie=None):
+    def _get_request(self, method=None, cookie=None, request_class=TestingHttpRequest):
```

Comment:
The code will be cleaner if you make the default `None` here and elsewhere, and then check for `None` below, similar to how `method` is done. In particular, you won't need to hard-code the default in more than one spot.

Reply:
Fixed. Thank you!